packages/wonder-blocks-core/components/clickable-behavior.js

```diff
@@ -383,11 +391,26 @@ export default class ClickableBehavior extends React.Component<
         // navigate.
         if (e.defaultPrevented) {
             shouldNavigate = false;
+            canSubmit = false;
         }
 
-        // Prevent navigation.
         e.preventDefault();
 
+        if (!href && type === "submit" && canSubmit) {
+            let target = e.currentTarget;
+            while (target) {
+                if (target instanceof window.HTMLFormElement) {
+                    const event = new window.Event("submit");
+                    target.dispatchEvent(event);
```

Comment:
Maybe it would be good adding a test with nested forms.

Reply:
That's a good idea.  I assume it's only supposed to trigger the innermost form.

Reply:
Actually, according to https://stackoverflow.com/questions/379610/can-you-nest-html-forms nested forms is not allowed.